src/components/MapView/Analyser/util.ts

```diff
@@ -0,0 +1,20 @@
+export const extractPropsFromURL = (path: string) => {
```

Comment:
You should specify the return type of this to be the AnalysisForm so we can ensure this always grabs everything needed

Reply:
Done